templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
+companies do. At a company, users are on their company's chat all day. In a
+part-time working group or community, users might check the chat less than
+once a week. In such a group, a good chat system can be the difference
+between robust community and non-existence.
```

Comment:
In such a group, making it easy for someone to participate effectively when they don't have time to read anything can be the difference between a robust, growing community and one that stagnates.